test-harness/specs/E2E-Accept-Content-Negotiation.complex.oas3.txt

```diff
@@ -0,0 +1,35 @@
+====test====
+When I send a request to an operation with Accept header containing multiple types.
+And this operation can produce a response that matches nth+1 content type
```

Comment:
hm.. isn't this operation only producing `application/json`?

Reply:
Yeah indeed, this text is not exactly clear to me

Reply:
Updated description to be more elaborative. Is it ok now?

Reply:
For sure, it's way better now, thanks!